pkgs/by-name/ha/hap-py/package.nix

```
@@ -0,0 +1,63 @@
+{ pkgs, stdenv, fetchFromGitHub, boost, zlib, lib, makeWrapper }:
+
+let
+  # Bcftools needs perl
+  runtime = with pkgs; [ bcftools htslib my-python perl samtools ];
+  my-python-packages = p: with p; [
+    bx-python
+    pysam
+    cython
+    pandas
+    psutil
+    nose
+    scipy
+  ];
+  my-python = pkgs.python3.withPackages my-python-packages;
```

```suggestion
  python = pkgs.python3.withPackages (p: with p; [
    bx-python
    pysam
    pandas
    psutil
    scipy
  ]);
```

nose is a test dependency usually and cython a compile time only dep